gnucash/import-export/import-main-matcher.c

```diff
@@ -72,6 +74,9 @@ struct _main_matcher_info
     GtkWidget         *reconcile_after_close;
     gboolean add_toggled;   // flag to indicate that add has been toggled to stop selection
     gint id;
+    GSList* temp_trans_list; // Temporary list of imported transactions
+    GHashTable* id_hash;     // Hash table, per account, of list of transaction IDs.
```

Comment:
Could this be renamed `acct_hash` both here and as the arg to `gnc_import_exists_online_id` to make it clearer that `id_hash` used in `collect_trans_online` is `new_hash` from `gnc_import_exists_online_id`?

Reply:
How about acct_id_hash, since it's a hash of id, per account?

Reply:
Sure. The main thing is that it's a different name.